lib/test.js

```diff
@@ -32,41 +28,46 @@ function sink ({ destroyOnError = false, emitErrorEvent = false } = {}) {
 
 /**
  * Check if the chunk is equal to the expected value.
- * @param {object} chunk: The chunk to be tested
- * @param {object} expected: The expected value to be tested
- * @param {assert.deepStrictEqual} is: The assert function to be used.
- * @throws If the expected value is not equal to the chunk value
+ *
+ * @param {object} chunk The chunk to be tested.
+ * @param {object} expected The expected value to be tested.
+ * @param {nodeAssert.deepStrictEqual} assert The assert function to be used.
```

Comment:
`{nodeAssert.deepStrictEqual}` is not a valid type. See https://jsdoc.app/tags-type. It should be `{function}`.